src/test/scala/ai/privado/languageEngine/go/audit/DataElementDiscoveryTestBase.scala

```diff
@@ -33,6 +36,9 @@ abstract class DataElementDiscoveryTestBase extends AnyWordSpec with Matchers wi
       .withOutputPath(outputDir.pathAsString)
     val goSrc2Cpg = new GoSrc2Cpg(None)
     val xtocpg = goSrc2Cpg.createCpg(config).map { cpg =>
+      new SQLQueryParser(cpg).createAndApply()
```

Comment:
change it to new test suite.

Reply:
Context: I tried using `GoFrontendTestSuite` but that does not seem to work as the text fixture does not accept `TaggerCache` from outside which is problematic since it fails any `sourceRuleIdMap` tests. Based on our discussion, here's what I've done:

1. Reverted the commit that utilised `*FrontendTestSuite`, and,
2. Will open a ticket internally.

Reply:
Will switch to newer one later.